guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode.adoc

```diff
@@ -0,0 +1,33 @@
+[id="setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode_{context}"]
+= Setting an alternative directory for remote execution jobs in pull mode
+
+By default, {Project} uses the `/run` directory on host for remote execution jobs in pull mode.
+
+If the `/run` directory on your host is mounted with the `noexec` flag, {Project} cannot execute remote execution job scripts in this directory.
+Use this procedure set a new directory for executing remote execution jobs in pull mode.
```

Comment:
```suggestion
By default, {Project} uses the `/run` directory on host for remote execution jobs in pull mode.
If the `/run` directory on your host is mounted with the `noexec` flag, {Project} cannot execute remote execution job scripts in this directory.
You can use the `yggdrasild` service to set an alternative directory for executing remote execution jobs in pull mode.
```
May I suggest slightly rewording the introduction to keep it in line with the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc)?

* A single paragraph feels more appropriate, because all the sentences are closely related (they represent the use case for the procedure, i.e. they're not just individual facts).
* Use consistent language ("alternative directory" throughout the module, rather than alternating between "alternative directory" and "new directory").
* Avoid self-referential language ("this procedure").

Reply:
Thank you, applied.